features/step_definitions/logging.rb

```diff
@@ -309,3 +309,26 @@
     raise "Get metrics failed with error, #{@result[:response]}"
   end
 end
+
+Given /^I open kibana console in a browser$/ do
```

Comment:
What is the difference with `Given I login to kibana logging web console`? Can one step be used for kibana instead of two?

Reply:
The step `Given I login to kibana logging web console` could be used for logging 3.x, for logging 4.x, the login has changed. I'm not sure if I could modify the step or not, so I added a new step for logging 4.x .

Reply:
the `master` branch is for 4.x and `v3` branch is for 3.x. So please change same step.